featuretools/tests/synthesis/test_deep_feature_synthesis.py

```diff
@@ -271,6 +271,16 @@ def test_make_groupby_features(es):
                               "CUM_SUM(value) by session_id"))
 
 
+def test_make_groupby_features_with_id(es):
+    dfs_obj = DeepFeatureSynthesis(target_entity_id='sessions',
+                                   entityset=es,
+                                   agg_primitives=[],
+                                   trans_primitives=[],
+                                   groupby_trans_primitives=['cum_count'])
+    features = dfs_obj.build_features()
+    assert (feature_with_name(features, "CUM_COUNT(customer_id) by customer_id"))
```

Comment:
can we also test for the `CUM_COUNT(id_1) by id_2` case

I think "customers.CUM_COUNT(cohort) by région_id" would satisfy this